### PR TITLE
Added clean agent procedure

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
- Fixed up TOC to fit new topic
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

### CODE REVIEW

**Comment by joel-hamill** [comment]
@cmaloney see `1.8/administration/remove-agent.md`

File: 1.8/administration/remove-agent.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+post_title: Removing an Agent
+menu_order: 3.2
+---
+
+To shut down and safely remove an agent node follow these steps.
+
+1.  [SSH](/docs/1.8/administration/access-node/sshcluster/) to the master node.
```
Comment (cmaloney) [inline 1.8/administration/remove-agent.md]: This needs to be sshing to the individual agent the user wants to de-register.

**Review by cmaloney — CHANGES_REQUESTED** [review]

**Comment by joel-hamill** [comment]
@sascala ready for merge

File: 1.9/administration/upgrading.md [hunk 2]
```diff
@@ -24,6 +25,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 - In CentOS or RedHat, install IP sets with this command (used in some IP detect scripts): `$ sudo yum install -y ipset`
 - You must be familiar with using `systemctl` and `journalctl` command line tools to review and monitor service status. Troubleshooting notes can be found at the end of this [document](#troubleshooting).
 - You must be familiar with the [Advanced DC/OS Installation Guide][advanced-install].
+- You should take a snapshot of ZooKeeper prior to upgrading. Marathon supports rollbacks, but does not support downgrades.
```
Comment (sascala) [inline 1.9/administration/upgrading.md]: Do we have instructions for this?

File: 1.9/administration/upgrading.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 - The Advanced Installation method is the _only_ recommended upgrade path for DC/OS. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation Guide][advanced-install] before proceeding.
 - The [VIP features](/docs/1.9/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/), added in DC/OS 1.8, require that ports 32768 - 65535 are open between all agent and master nodes for both TCP and UDP.
+- Overlay networks require Docker 1.11. For more information, see the [documentation](/docs/1.9/administration/overlay-networks/).
```
Comment (sascala) [inline 1.9/administration/upgrading.md]: Are overlay networks new? Or do you mean if you're upgrading with overlay networks? or both? More generally, what is the context?

**Review by sascala — COMMENTED** [review]
I had two questions, but otherwise LGTM

File: 1.9/administration/upgrading.md (hunk 3 above)
Comment (joel-hamill) [inline 1.9/administration/upgrading.md]: Not new, but as you're upgrading DCOS you might be tempted to use a different docker version, and the overlay networks feature.

**Pull request merged**